.ci/setup.sh

```diff
@@ -146,7 +146,7 @@ main() {
 	sync
 	sudo -E PATH=$PATH bash -c "echo 3 > /proc/sys/vm/drop_caches"
 
-	if [ "$ID" == rhel ]; then
+	if [ "$ID" == rhel ] && [ "$PLATFORM_ID" == "platform:el7" ] ; then
```

Comment:
v7 only? If so, please add a comment explaining why. Else could you instead use something like this?:

```bash
[[ "$PLATFORM_ID" =~ ^platform:el[0-9][0-9]*$ ]]
```

Reply:
So I cannot really find a good source on where and when `/proc/sys/fs/may_detach_mounts` is supported.
There's this [Red Hat bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1441737) that claims it was introduced in 7 (RHEL only, no CentOS), but it's not on my x86_64 laptop or my s390x dev instance (RHEL 8.4 and 8.3, respectively). It's also not in the [kernel docs](https://www.kernel.org/doc/html/latest/admin-guide/sysctl/fs.html). I don't really have a RHEL 7 instance handy to test with.